ts/boot/configureStoreAndPersistor.ts

```diff
@@ -62,7 +62,17 @@ const migrations: MigrationManifest = {
   // Adds messagesIdsByServiceId
   "2": (state: PersistedState) => {
     return addMessagesIdsByServiceId(state as PersistedGlobalState);
-  }
+  },
+
+  // Version 3
+  // we changes the organizations so we clear the entitie of organizations to force a reload
+  "3": (state: PersistedState): PersistedState =>
+    ({
+      ...state,
+      entities: {
+        organizations: {}
```

Comment:
Aren't you deleting all the other entities?

Reply:
Fixed in commit 1565112

Reply:
Can you try something like.

```
const clonedState = {...state} as PersistentGlobalState;
delete clonedState.entities.organizations;
return clonedState;
```

Reply:
What i mean is cast the state only once.

Reply:
Ok! I try

Reply:
@fpersico To use this code it is necessary to remove the Readonly attribute on EntitiesState and disable a tslint rule (no-object-mutation). What do you think? I proceed?

Reply:
No. Just cast the state once. But please check what @Undermaken said: "I tested the migration but an error is still living".